tools/PI/DevHome.PI/Models/AppRuntimeInfo.cs

```diff
@@ -82,4 +86,26 @@ public void CheckFrameworkTypes(string moduleName)
             }
         }
     }
+
+    public void CheckFrameworkTypes(ProcessModuleCollection modules)
+    {
+        foreach (ProcessModule module in modules)
+        {
+            CheckFrameworkTypes(module.ModuleName);
+        }
+
+        // Special-case for UWP apps:
+        // UWP will be set true if we found OneCoreUAPCommonProxyStub.dll, but this is also used
```

Comment:
>OneCoreUAPCommonProxyStub [](http://example.com/codeflow?start=44&length=25)

thats probably not a good proxy for UJWPness. There are tons of com class registrations in there that can be used by non-UWP apps.
I would think we need some combination of appcontainer + package identity to detect UWPness.

Reply:
Hmm... yes, and appcontainer/package ID won't help much either, too much overlap with winui/maui. I think I'll abandon this.